src/sidebar/components/NotebookView.js

```diff
@@ -111,6 +124,10 @@ function NotebookView({ loadAnnotationsService }) {
     }
   }, [paginationPage]);
 
+  const tooltip = `Show ${pendingUpdateCount} new/deleted/updated ${
```

Comment:
Sorry I'm just noticing this now and didn't catch it in an earlier pass. The title wording on this is a little bulky. I can see you pulled it from the `TopBar` component. How about:

"Show ${pendingUpdateCount} new or updated annotation[s]"

I think deletion counts as an update.

Reply:
Sure